assets/source/setup-guide/app/style.scss

```diff
@@ -345,40 +300,18 @@ $root: ".woocommerce-setup-guide";
 
 				.components-card__body {
 
-					> *:not(:last-child) {
-						margin-bottom: 16px;
-					}
-				}
-
-				#{$root} {
-
-					&__step {
-
-						&-status {
-							padding: 0;
-						}
+					> *:not(:first-child) {
+						margin-bottom: var(--main-gap);
 					}
-				}
-
-				.is-verified {
 
-					> * {
-						margin: 8px 0;
-					}
-				}
-
-				.has-idle {
-
-					#{$root} {
-
-						&__step-status__icon {
-							display: none;
-						}
+					> *:first-child {
+						margin-bottom: $grid-unit-10;
 					}
 				}
 
-				.claim-error {
-					color: var(--wc-red);
+				.components-notice.is-error {
+					margin: 0;
+					background-color: #f8ebea;
```

Comment:
![image](https://user-images.githubusercontent.com/17420811/144958475-2dbdb9d8-b9e2-453e-b153-831c29589f18.png)

The first one is the default background color of `<Notice>` component. Since the difference is very obvious, I thought the customization was needed here.

Reply:
@shivanarrthine what do you think? Should we just stick with the default background color of the component?

Reply:
Thanks for raising this. Let's stick to using the default background color for the component.

Reply:
Updated by https://github.com/woocommerce/pinterest-for-woocommerce/pull/286/commits/d8a65906258a6bb0a4849ac7b829b94cb7ad9751.

![image](https://user-images.githubusercontent.com/17420811/145138616-041246a8-e8f9-4980-ab21-0b71ff2c3e04.png)